handlers/anteHandler_test.go

```diff
@@ -34,21 +34,21 @@ type inputUTXO struct {
 type conn struct{}
 
 // all deposits should be in an amount of 10eth owner by addr(defined above)
-func (p conn) GetDeposit(nonce *big.Int) (plasma.Deposit, *big.Int, bool) {
+func (p conn) GetDeposit(tmBlock *big.Int, nonce *big.Int) (plasma.Deposit, *big.Int, bool) {
```

Comment:
Wanna change this to plasmaBlock so if someone different looks at this code they don't get too confused

Reply:
+1